src/test/shared/awsClientBuilder.test.ts

```diff
@@ -9,7 +9,7 @@ import * as assert from 'assert'
 import { Service } from 'aws-sdk'
 import { ServiceConfigurationOptions } from 'aws-sdk/lib/service'
 import * as vscode from 'vscode'
-import { AWSClientBuilder } from '../../shared/awsClientBuilder'
+import { DefaultAWSClientBuilder } from '../../shared/awsClientBuilder'
 import { AwsContext, ContextChangeEventsArgs } from '../../shared/awsContext'
 
 describe('AwsClientBuilder', () => {
```

Comment:
should this be describe `DefaultAWSClientBuilder` now instead? If so, consider renaming the file too.